get-started/get-started.adoc

```diff
@@ -19,105 +26,4 @@ Take your first steps in becoming an expert Neo4j developer.
 2. link:/developer/cypher[Learn to Create and Query Data] and link:/online-course[Take the Free Online Course]
 3. link:/developer/working-with-data/guide-importing-data-and-etl[Import Your Data]
 4. link:/developer/language-guides[Build an Application]
-5. Get Help on http://stackoverflow.com/questions/tagged/neo4j[StackOverflow], http://neo4j.com/slack[Slack], or http://neo4j.com/contact-us/[by contacting us]
```

Comment:
I'm not fond of removing the welcoming/guiding sentences and sections guiding people to the different sections of the site. 
This creates in my view an anemic page.
Also removing the video will make this page much less welcoming.

Listing the supported programming language (drivers) as a compact table was intended to have everyone feel supported in their stack and being immediately able to get to the relevant information.

Same for highlighting and introducing GraphAcademy and the other information sources.

Reply:
We're trying to make things more prescriptive and readable.  This means cutting down on duplicate information and tons of paths to reach the same information.

As it stands, this page has duplicate information all over it instead of just guiding to the relevant other pages. 

ie, we have a page on "What is a graph database?"   I can understand having a high level summary on this page, but then we should link to that page and that page only below this summary-- not 5 or 6 links deep into that content.

similar for the table of languages -- we should have that on the languages page and link to there.  the getting started page shouldn't be the authoritative source for this.

also for things like graphacademy -- we shouldn't have a bunch of deep links.  Let's get rid of those links, add a few sentences, and guide into the graphacademy page. 

i feel like we're torturing jennifer with our different opinions here.  let's chat f2f (on hangout) and reconcile.